Detectors/MUON/MCH/Raw/Decoder/src/BareElinkDecoder.h

```diff
@@ -301,7 +301,7 @@ void BareElinkDecoder<CHARGESUM>::handleReadSample()
   }
   oneLess10BitWord();
   if (mNofSamples) {
-    handleReadData();
+    changeToReadingData();
```

Comment:
This is how the `mNofSamples` and `mNof10BitsWordsToRead` are decremented in the original code:
```
BareElinkDecoder::handleReadSample : mNofSamples: 5, mNof10BitsWordsToRead: 5
BareElinkDecoder::handleReadSample : mNofSamples: 4, mNof10BitsWordsToRead: 4
BareElinkDecoder::handleReadSample : mNofSamples: 3, mNof10BitsWordsToRead: 3
BareElinkDecoder::handleReadSample : mNofSamples: 2, mNof10BitsWordsToRead: 2
BareElinkDecoder::handleReadSample : mNofSamples: 1, mNof10BitsWordsToRead: 1
BareElinkDecoder::handleReadSample : end-of-cluster found, mNof10BitsWordsToRead: 0
```